modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -165,6 +165,27 @@ inline void copyFromIE(const IE::Blob::Ptr &blob, MatType &mat) {
     }
 }
 
+// (taken from videoio module)
+static cv::Mutex& getInitializationMutex()
```

Comment:
why not `std::mutex`?

Reply:
See: https://github.com/opencv/opencv/blob/4.0.0/modules/core/include/opencv2/core/utility.hpp#L697